Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:stretch
+
+ENV LANG=C.UTF-8
+
+RUN set -x \
+  && apt-get update \
```

Comment:
with recent apt, you can just run "apt" instead of "apt-get".

Reply:
debian/debian based distros recommend not using `apt` in scripting and it won't throw errors, but will throw a bunch of text to `stderr` if you use `apt` instead of `apt-get`. Feel free to try in a small bash script if you're interested. I recommend keeping `apt-get`.

Reply:
interesting, didn't know that recommendation.